cernopendata/modules/fixtures/data/docs/delphi-about/delphi-about.json

```diff
@@ -0,0 +1,25 @@
+[
+  {
```

Comment:
One more suggestion (feel free to ignore it if you don't agree). At the moment, this PR creates two documents. What about adding a third one, with the `news` of the opening of the data? The document would be very short, maybe just with the links to the `about` and `getting started` for more info.

That one will then be displayed in the [news section](https://opendata.cern.ch/#news).

Reply:
I have added a news section now. Please take a look if I got that one right.

Reply:
Looks good, thanks. One minor comment, and that will be treated independently. Resolving this one